src/lightning_app/testing/testing.py

```diff
@@ -210,7 +208,9 @@ def run_cli(args) -> Generator:
 
 @requires("playwright")
 @contextmanager
-def run_app_in_cloud(app_folder: str, app_name: str = "app.py", extra_args: [str] = []) -> Generator:
+def run_app_in_cloud(
+    app_folder: str, app_name: str = "app.py", extra_args: List[str] = [], debug: bool = True
```

Comment:
```suggestion
    app_folder: str, app_name: str = "app.py", extra_args: Sequence[str] = tuple(), debug: bool = True
```
this is not imutable